core/src/main/java/org/apache/accumulo/core/client/admin/TableOperations.java

```diff
@@ -957,6 +960,23 @@ IteratorSetting getIteratorSetting(String tableName, String name, IteratorScope
   Map<String,EnumSet<IteratorScope>> listIterators(String tableName)
       throws AccumuloSecurityException, AccumuloException, TableNotFoundException;
 
+  /**
+   * Create a stack of iterators from what is in the provided table properties and with the scope.
+   * The specified source will be the bottom of the stack and the top SortedKeyValueIterator on the
+   * stack will be returned.
+   *
+   * @param tableProps
+   *          the properties of the table where the iterators are configured
+   * @param scope
+   *          the scope of the iterators
+   * @param source
+   *          the iterator at the start of the stack
+   * @return the SortedKeyValueIterator at the top of the stack
+   * @since 2.1
+   */
+  SortedKeyValueIterator<Key,Value> loadIterators(Iterable<Entry<String,String>> tableProps,
```

Comment:
Since this a client side operation only and does nothing to a table, seems like it should be placed somwhere else.  Maybe a static method in the iterators package.

Reply:
It was originally in IteratorUtil so we could just put it there but with the API friendly types... assuming we want to have this functionality as an API.